demisto_sdk/commands/common/errors.py

```diff
@@ -67,9 +67,11 @@
     "multiple_default_arg": "IN134",
     "missing_output_context": "IN136",
     "invalid_integration_parameters_display_name": "IN135",
+    "is_valid_integration_file_path": "IN136",
```

Comment:
I think @JasBeilin is using IN136 if you merge from master you should see that.

Reply:
done